src/nitclk/metadata/functions.py

```diff
@@ -19,6 +19,7 @@
                     'mechanism': 'TBD',
                     'value': 'TBD'
                 },
+                'type_in_documentation': 'list of int, list of nimi-python Session class, list of SessionReference',
```

Comment:
I think we can remove `int`.
The other two are still up for debate.

"list of nimi-python Session or nitclk.SessionReference"

Reply:
Until #1042, this is the correct list.

These would change based on the decision and change for that issue.